support/mock-data/Terms/count/Genetics/HealthProfessional/en/count.json

```diff
@@ -1 +1 @@
-232
+245
```

Comment:
Why the need to change the number?

Reply:
That's the number that actually gets returned from the API, and also stated in the acceptance criteria. Since there was no other reference to this in the code felt it was appropriate to update this as well.